client/modules/workspace/src/JobStatusNav/JobStatusNav.module.css

```
@@ -6,3 +6,16 @@
 .badge {
   margin: 0 10px;
 }
+
+.root:hover,
+.root:focus,
+.highlighted-row,
+.highlighted-row:hover,
+.highlighted-row:focus {
```

This code is good. I merely offer new syntax in context.

```suggestion
root:is(:hover, :focus),
.highlighted-row:is(:default, :hover, :focus) {
```

I have used [`:is(…)`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) enough to know this works. The [`:default`](https://developer.mozilla.org/en-US/docs/Web/CSS/:default) is new to me, but widely available.

---

I removed proposed use of [`:default`](https://developer.mozilla.org/en-US/docs/Web/CSS/:default), which is stated to be only applicable to form elements. The [:is(…)](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) suggestion is valid.